### PR TITLE
DOCS-4674: Tweak data directory size FAQ

### PR DESCRIPTION
**Pull request by i80and**

### CODE REVIEW

File: source/faq/storage.txt [hunk 1]
```diff
@@ -116,63 +116,64 @@ The data files in your data directory, which is the :file:`/data/db`
...
-  .. warning::
-     :dbcommand:`repairDatabase` requires enough free disk space to
-     hold both the old and new database files while the repair is
-     running. Be aware that :dbcommand:`repairDatabase` will block
-     all other operations and may take a long time to complete.
+Preallocated data files
+~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB preallocates its data files, in part to avoid file system fragmentation.
+The first file that :program:`mongod` creates, ``<databasename>.1``, will be 64
```
Comment (kay-kim) [inline source/faq/storage.txt]: is it ``<databasename>.1`` or ``<databasename>.0``

File: source/faq/storage.txt [hunk 2]
```diff
@@ -116,63 +116,64 @@ The data files in your data directory, which is the :file:`/data/db`
...
+megabytes. The second, ``<databasename>.2``, will be 128 megabytes; and so on,
+up to 2 gigabytes.
+
+The proportion of the preallocated files that is unused is typically small.
```
Comment (kay-kim) [inline source/faq/storage.txt]: This paragraph, read in context of the previous paragraph, has the following logic trail:
1. MongoDB prealllocates data files.  For a database, the first data file  = 64 Megabytes.
2. The proportion of the preallocated files that is unused is typically small.
3. However, smallfiles reduce size of preallocated files.  

2 needs qualifications for it to be consistent with 1.
How is 3 a "however" of 2 ?

File: source/faq/storage.txt (hunk 1 above)
Comment (i80and) [inline source/faq/storage.txt]: Oh, it's absolutely <databasename>.0.  Good catch, thanks!

File: source/faq/storage.txt [hunk 3]
```diff
@@ -116,63 +116,64 @@ The data files in your data directory, which is the :file:`/data/db`
...
-     hold both the old and new database files while the repair is
-     running. Be aware that :dbcommand:`repairDatabase` will block
-     all other operations and may take a long time to complete.
+Preallocated data files
+~~~~~~~~~~~~~~~~~~~~~~~
+
+MongoDB preallocates its data files, in part to avoid file system fragmentation.
+The first file that :program:`mongod` creates, ``<databasename>.0``, will be 64
+megabytes. The second, ``<databasename>.1``, will be 128 megabytes; and so on,
+up to 2 gigabytes. The :setting:`storage.mmapv1.smallFiles` option will reduce the
```
Comment (kay-kim) [inline source/faq/storage.txt]: almost.  But the second sentence makes it seems like we stop pre-allocating data files once it allocates the first 2 gigabyte data file.

**Comment by mpaik** [comment]
@kay-kim is this ready for merge?

**Comment by kay-kim** [comment]
yup

**Comment by mpaik** [comment]
Merged

**Pull request closed without merging**